.ebextensions/loadbalancer.config

```diff
@@ -25,7 +25,7 @@ option_settings:
   # https://docs.aws.amazon.com/elasticbeanstalk/latest/dg/environments-cfg-alb.html#environments-cfg-alb-namespaces
 
   aws:elbv2:listener:443:
-    SSLCertificateArns: arn:aws:acm:us-east-1:100225593120:certificate/70263776-5e62-4454-9997-bb7e560d7c63
+    SSLCertificateArns: arn:aws:acm:us-east-1:100225593120:certificate/047a56a2-09dd-4857-9f28-32d23650d4da
```

Comment:
PS @abecerra this will need to be done in Mati and the FMS.

Reply:
Switches from our own manually updated cert to amazons auto renewed cert.

Reply:
> @mluypaert will this take effect or are we going to need to delete the ENV?

Not sure, this will be a load balancer change so we'll need to see if EB can handle this as an update. I would expect this not to be a problem as it it a single property on the load balancer, nothing would change from the application instance's point of view. The deployment GH actions workflow will fail with clear errors otherwise, in which case we will know we need to delete and recreate.

Reply:
@mluypaert ok... I am also thinking that maybe we can go and just update them manually in the LB's rather than deleting the ENV